recipes/ghc/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+export LDFLAGS="-L${PREFIX}/lib"
+export CPPFLAGS="-I${PREFIX}/include"
+ls
+if [ `uname` == Darwin ]
+then
+    export DYLD_LIBRARY_PATH="${PREFIX}/lib"
```

Comment:
Why is this needed?

Reply:
Mac variant of library path, also to ensure linking against conda-forge libraries